### PR TITLE
Feature/pdct 1571 render multiple geographies in frontend

### PR DESCRIPTION
**Pull request by katybaulch**
# What's changed

Render multiple geographies in the frontend.

## Proposed version

Please select the option below that is most relevant from the list below. This
will be used to generate the next tag version name during auto-tagging.

- [ ] Skip auto-tagging
- [x] Patch
- [ ] Minor version
- [ ] Major version

Visit the [Semver website](https://semver.org/#summary) to understand the
difference between `MAJOR`, `MINOR`, and `PATCH` versions.

Notes:

- If none of these options are selected, auto-tagging will fail
- Where multiple options are selected, the most senior option ticked will be
  used -- e.g. Major > Minor > Patch
- If you are selecting the version in the list above using the textbox, make
  sure your selected option is marked `[x]` with no spaces in between the
  brackets and the `x`

### CODE REVIEW

File: src/components/CountriesLink.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  geographies: string[];
+  countries: TGeography[];
+};
+
+export const CountriesLink: FC<TCountriesLink> = ({ geographies, countries }) => (
+  <>
+    {geographies?.map(
+      (geography, index) =>
+        !isSystemGeo(geography) && (
+          <span key={index} className="flex gap-1">
```
Comment (PatFawbertMills) [inline src/components/CountriesLink.tsx:17]: Use the geo as the key, index as key should only be for situations where we cannot be sure the values in the collection will be distinct.

File: src/components/CountriesLink.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import { getCountryName } from "@helpers/getCountryFields";
+import { isSystemGeo } from "@utils/isSystemGeo";
+import { TGeography } from "@types";
+
+type TCountriesLink = {
+  geographies: string[];
+  countries: TGeography[];
+};
+
+export const CountriesLink: FC<TCountriesLink> = ({ geographies, countries }) => (
```
Comment (PatFawbertMills) [inline src/components/CountriesLink.tsx:12]: This component doesn't need to be a FunctionalComponent (and I think this is a legacy feature of react now anyway, reserved for when we need to use `children` as a prop)

**Review by PatFawbertMills — CHANGES_REQUESTED** [review]
Couple of v minor react-y things

File: src/components/CountriesLink.tsx (hunk 2 above)
Comment (katybaulch) [reply src/components/CountriesLink.tsx:12]: Ahh okay - my bad blindly copying from other parts of the codebase! Thanks for letting me know. Think I've updated accordinly - but let me know if I've still got it wrong

**Comment by katybaulch** [comment]
Driveby fix for a bug where XAB geographies are clickable in the breadcrumb
image_720

**Review by PatFawbertMills — APPROVED** [review]
LGTM

**Comment by katybaulch** [comment]
> what happens in the breadcrumb when there are multiple geos? I can see in the code here that it looks like we don’t display anything?
> 
> ```typescript
> const breadcrumbGeography = family.geographies && family.geographies.length > 1 ? null : { label: geoName, href: `/geographies/${geoSlug}` };
> ```

it sets the breadcrumbgeo value to null, and it shows this (which is consistent with Conor's designs)

image_720

**Pull request merged**